server/bootstrap/build.gradle

```diff
@@ -16,10 +16,12 @@ sourceSets {
 
 dependencies
         {
-            implementation "org.apache.tomcat:tomcat-api:${project.apacheTomcatVersion}"
-            implementation "org.apache.tomcat:tomcat-catalina:${project.apacheTomcatVersion}"
-            implementation "org.apache.tomcat:tomcat-juli:${project.apacheTomcatVersion}"
-            implementation "org.apache.tomcat:tomcat-util:${project.apacheTomcatVersion}"
+            def tomcatVersion = BuildUtils.useEmbeddedTomcat(project) ? project.springBootTomcatVersion : project.apacheTomcatVersion
```

Comment:
I assume you tried leaving this as the 8.5 version and ran into problems?

Reply:
Yes, https://teamcity.labkey.org/repository/download/LabKey_Trunk_Internal_Embedded_SampleManagerEmbeddedPostgres/1243429:id/embeddedTomcatLogs.tar.gz!/embeddedTomcat.log